public/v4/apps/poste-io.yml

```diff
@@ -1,52 +1,143 @@
+            7. Go to your application navigating to `http://$$cap_appname.$$cap_root_domain/admin/domain`
+            8. Click **Show** on `$$cap_POSTE_HOSTNAME`
+            9. In the line **DKIM Key** click on **Create New Key** and copy the generated value
+            10. Go back to the DNS settings
+            11. Create **TXT** record and insert copied value (starting with `k=rsa...`)
+            12. Verify the DNS configuration using tools like MXToolbox (`https://mxtoolbox.com`)
 
-            The deployment can take up to 2 minutes.
-    displayName: Poste.io
-    isOfficial: true
-    description: Poste.io - complete mailserver built in one container
-    documentation: Read the poste.io documentation @ https://poste.io/doc
+            If all your checks are green you are ready to create mailboxes.
+
+            Enjoy your self-hosted mail server!
+    variables:
+        - id: $$cap_POSTE_VERSION
+          label: General | Version
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/analogic/poste.io/tags
+          defaultValue: '2'
+          validRegex: /.{1,}/
+        - id: $$cap_POSTE_TZ
+          label: General | Timezone
+          description: Timezone for the application, find yours at https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+          defaultValue: UTC
+          validRegex: /.{1,}/
+        - id: $$cap_POSTE_DISABLE_CLAMAV
+          label: Features | Disable ClamAV
+          description: Whether to disable ClamAV anti-virus. Useful for low memory usage. Set value `TRUE` to disable.
+        - id: $$cap_POSTE_DISABLE_RSPAMD
+          label: Features | Disable Rspamd
+          description: Whether to disable Rspamd spam filter. Useful for low memory usage. Set value `TRUE` to disable.
+        - id: $$cap_POSTE_DISABLE_ROUNDCUBE
+          label: Features | Disable Roundcube
+          description: Whether to disable Roundcube webmail. Set value `TRUE` to disable.
+        - id: $$cap_POSTE_HOSTNAME
+          label: Networking | Hostname
+          description: Hostname of your mail server. **This is required**. It is usually `mail.mydomain.com`.
+          validRegex: /.{1,}/
+        - id: $$cap_POSTE_HTTPS
+          label: Networking | Use HTTPS
+          description: >-
+              Whether to redirect to encrypted HTTP.
+              It's recommended to leave it to `OFF` since CapRover acts as reverse proxy and handles HTTPS.
+              If you want to enable it, remove the value and leave it blank and then to change the configuration and ports accordingly.
+          defaultValue: 'OFF'
+        - id: $$cap_POSTE_PORT_SMTP_UNENCRYPTED
+          label: Networking | Unencrypted SMTP Port
+          description: Port for unencrypted mail submission. Should not be changed.
```

Comment:
If they should not be changed, is there any benefit in keeping them as variables? Why not hardcode them for the sake of simplicity of the file and future maintenance?

Reply:
Agree, reverted to hardcoded values.
Thanks.